spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
Alright, on it. =)

Hopefully that [flaky issue](https://github.com/atom/tree-view/pull/859#issuecomment-234780696) with the random spec-failure won't bite me...